deploy/supervisor/helpers.lib.yaml

```diff
@@ -53,17 +53,11 @@ _: #@ template.replace(data.values.custom_labels)
 #@       "apiService": defaultResourceNameWithSuffix("api"),
 #@     },
 #@     "labels": labels(),
-#@     "insecureAcceptExternalUnencryptedHttpRequests": data.values.deprecated_insecure_accept_external_unencrypted_http_requests
```

Comment:
Anything in this configmap must also have Golang code associated with it, right? Shall we find and remove that code too?

Reply:
Done!